systemjs/basic-import-map/mercury.js

```diff
@@ -0,0 +1,11 @@
+System.register([], function (_export, _context) {
+  return {
+    execute: function() {
+      _export('doAlert', doAlert);
+
+      function doAlert() {
+        alert("Mercury is the planet nearest to the sun.");
+      }
+    }
+  }
+})
```

Comment:
It would be worth sharing how to create System.register modules in due course too. Ideally the workflow could output these from ESM.

Reply:
yeah i plan on adding that in future examples.